### DISCUSSION TITLE
Merging is blocked although deployment succeeds

### DISCUSSION THREAD

**Discussion by jack-wombat** [vercel/community#4834] [Help]
### Summary

This is happening for our dev twice in a row. The deploment is successful on the "Preview" enviornment. But github is still blocking it with `Missing successful Preview deployment. `. What can I do?

![image](https://github.com/vercel/community/assets/97011001/a486545d-5b1b-4074-84ea-eb1cdb4ece4c)

### Steps to Reproduce

This seems to be working before. I connect the github-vercel integration through vercel website. And it creates both the Preview and Production environments.

**Comment by amyegan** [comment]
Hi @jack-wombat! This looks similar to https://github.com/vercel/community/discussions/3215#discussioncomment-6969235. In their case, the problem was caused be a mismatch in deployment environment setting. Please double check your GitHub repo settings to make sure the deployment environment being checked matches the deployment environment being used.

**Reply by jack-wombat** [reply]
You can see that in the first modal in the screenshot. It spells "Preview" correctly. Is there any way I can debug this further myself? I'm really stuck here.
